src/features/user/MFV2/ForestProgress/ForestProgressItem.tsx

```diff
@@ -0,0 +1,56 @@
+import styles from './ForestProgress.module.scss';
+import { getAchievedTarget } from '../../../../utils/myForestV2Utils';
+import { useMemo } from 'react';
+import { useRouter } from 'next/router';
+import EditButton from './microComponents/EditButton';
+import ProgressData from './microComponents/ProgressData';
+
+export type DataType = 'treesPlanted' | 'areaRestored' | 'areaConserved';
+
+export interface ForestProgressItemProps {
+  handleOpen: () => void;
+  dataType: DataType;
+  target: number;
+  gift: number;
+  personal: number;
+}
+
+const ForestProgressItem = ({
+  handleOpen,
+  dataType,
+  target,
+  gift,
+  personal,
+}: ForestProgressItemProps) => {
+  const { asPath } = useRouter();
+
+  const _getAchievedTarget = useMemo(
+    () => getAchievedTarget(target, gift, personal),
+    [target, gift, personal]
+  );
+
+  return (
+    <div className={`${styles.progressMainContainer} ${dataType}`}>
+      {asPath === '/en/profile/mfv2' && (
```

Comment:
I'd suggest not using path comparisons for logic. Paths can be changed, and then we need to remember to update the logic everywhere.

Use the profile page type to determine whether to show the Edit buttons, similar to how it is passed for `CommunityContributions`, `MyContributions` etc. in `ProfileLayout` and `PublicProfileLayout`